cookbook/workflow/homestead.rst

```diff
@@ -0,0 +1,66 @@
+.. index:: Vagrant, Homestead
+
+Using Symfony with Homestead/Vagrant
+====================================
+
+In order to develop a Symfony application, you might want to use a virtual
+development environment instead of the built-in server or WAMP/LAMP. Homestead_
+is an easy-to-use Vagrant_ box to get a virtual environment up and running
+quickly.
```

Comment:
I'm worried about this because people usually brag about Vagrant advantages but forget its drawbacks. Besides being much much much slower than the native application, it causes lots of other minor issues/troubles/misunderstandings (like these [Symfony issues related (somehow) to Vagrant](https://github.com/symfony/symfony/issues?utf8=%E2%9C%93&q=is%3Aissue+vagrant+)).

I'm not saying that Vagrant is bad. But I'd like to state clear to readers that using Vagrant will make your Symfony run very slow and it may cause other (minor) troubles totally unrelated to Symfony.

And about linking to third-party resources, **I think I was wrong**, I apologize for that and I think we should start linking to trustworthy third-party resources, like Benjamin's blog.